install/error-handling.sh

```diff
@@ -1,10 +1,34 @@
 echo "${_group}Setting up error handling ..."
 
+export SENTRY_DSN='https://afa81b4b7c634e318b0ec138abb2645f@self-hosted.getsentry.net/2'
+export SENTRY_ORG=sentry
+export SENTRY_PROJECT=dogfooding
+
+if [[ -f .reporterrors ]]; then
+  if [[ "$(cat .reporterrors)" == "yes" ]]; then
+    export REPORT_ERRORS=1
+  else
+    export REPORT_ERRORS=0
+  fi
+else
+  echo "Would you like to opt-in to error monitoring for the Sentry installer?"
+  echo "This helps us catch and fix errors when installing Sentry."
+  echo "We may retain your OS username, IP address, and installer log, for 30 days."
```

Comment:
tiny: `We may retain your OS username, IP address, and installer log for 30 days.`

Reply:
Good use case for [GitHub suggestions](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request), @hubertdeng123. ;)

```suggestion
  echo "We may retain your OS username, IP address, and installer log for 30 days."
```